### ISSUE TITLE
[BUG] 每次运行 `zhlint --fix` 时都会在文件最后添加一个空行

### ISSUE THREAD

**Issue by zyxkad** [Jinjiang/zhlint#122]
如题：每次运行 `zhlint --fix` 时都会在文件最后添加一个空行

<img width="707" alt="Screenshot 2023-10-20 at 10 40 52 AM" src="https://github.com/Jinjiang/zhlint/assets/91082411/6c7d5409-5ad4-40ee-842e-cdaaa9a90abb">

执行过程:
```sh
$ pnpm exec zhlint 'docs/**/*.md' --fix
[start] docs/analyzer.md
No error found.
[fixed] docs/analyzer.md
```

配置文件:
```json
{
  "preset": "default",
  "rules": {
    "unifiedPunctuation": "simplified"
  }
}
```

**Comment by zyxkad** [comment]
preset写错了，没事了
